WordPress/Classes/ViewRelated/Blog/Blog Dashboard/DashboardCard.swift

```diff
@@ -111,6 +116,15 @@ enum DashboardCard: String, CaseIterable {
         }
     }
 
+    /// A list of cards that can be shown/hidden on a "Personalize Home Tab" screen.
+    static let personalizableCards: [DashboardCard] = [
+        .todaysStats,
+        .draftPosts,
+        .scheduledPosts,
+        .blaze,
+        .prompts
+    ]
+
```

Comment:
@kean Sorry for being late to the party here, but what do you think about doing something like this?

```suggestion
    // Boolean indicating whether the DashboardCard can be shown/hidden from the "Personalize Home Tab" screen.
    var isPersonalizable: Bool {
        switch self {
        // add a case for each card
        }
    }
    
    static var personalizableCards: [DashboardCard] {
        DashboardCard.allCases.filter({ $0.isPersonalizable })
    }
```

This will help us avoid not adding future cards to the personalize screen if applicable, especially for developers unaware of these new changes.

Reply:
I like that. The `personalizableCards` were initially defined as an array because the order was important – it didn't match the cards' order on the Dashboard. But I just confirmed late last week that it could and should match.

I'm planning to generate this array based on whether the `BlogDashboardPersonalizationService` defines a key for a card: 

```swift
static var personalizableCards: [DashboardCard] {
    DashboardCard.allCases.filter {
        BlogDashboardPersonalizationService.makeKey(for: $0 != nil)
    }
}
```

Reply:
This will achieve the same thing I had in mind 👍 
The only issue is we'll have to make `BlogDashboardPersonalizationService.makeKey` public for this reason alone. It might make more sense to move `personalizableCards` to `BlogDashboardPersonalizationService` at that point. This way, we'll have all the personalization logic in one place. Just something to think about 👍

Reply:
My personal recommendation would be to change `makeKey()` into a computed property `DashboardCard.personalizeKey` defined in a private extension in `BlogDashboardPersonalizationService.swift`. This will enhance the verbosity of the code.
And move `personalizableCards` to `BlogDashboardPersonalizationService`.

Reply:
I opened a draft PR with this change: #20466.

> It might make more sense to move personalizableCards to BlogDashboardPersonalizationService at that point. 

Makes total sense, thanks!